network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py

```diff
@@ -27,3 +27,22 @@ def extended_yes_no(value):
     if value == 'No':
         return gettext('No')
     return value
+
+
+@register.filter
+def track_record(value):
+    """
+    effects localization for company track records. While it might
+    seem easier to just return gettext(value), we want to be explicit
+    about the possible options, and the context in which to apply
+    this tag, rather than a generic "localize" tag.
+    """
+    if value == 'Great':
+        return gettext('Great')
```

Comment:
@TheoChevalier does `gettext` allow passing an extra value to indicate what the translation context is?

Reply:
Yes! You need to use `pgettext()` for this, here’s an example: https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/buyersguide/views.py#L143-L145